src/server/scripts/Commands/cs_event.cpp

```diff
@@ -110,13 +101,13 @@ class event_commandscript : public CommandScript
         }
 
         GameEventMgr::ActiveEvents const& activeEvents = sGameEventMgr->GetActiveEventList();
-        bool active = activeEvents.find(eventId) != activeEvents.end();
+        bool active = activeEvents.find(*eventId) != activeEvents.end();
```

Comment:
yeah like same question for all of these, `eventId` implicitly converts to `uint32`

Reply:
```
../src/server/scripts/Commands/cs_event.cpp:104:48: error: no matching function for call to ‘std::set<short unsigned int>::find(const Trinity::ChatCommands::Variant<Trinity::ChatCommands::Hyperlink<Trinity::Hyperlinks::LinkTags::gameevent>, unsigned int>&) const’
```